views/elements/current_and_next.tmpl

```diff
@@ -0,0 +1,20 @@
+{{define "current_and_next"}}
+
+	{{if .Current}}
```

Comment:
Lots of whitespace here, which is not being removed ( https://golang.org/doc/go1.6#template ) perhaps some can be trimmed? or stripped accordingly?